rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -172,22 +172,13 @@ def _check_attribute_vocabulary(self, attribute: Text) -> bool:
         except (AttributeError, KeyError):
             return False
 
-    def _get_attribute_vocabulary(self, attribute: Text) -> Optional[Dict[Text, int]]:
-        """Get trained vocabulary from attribute's count vectorizer"""
+    def _get_attribute_vocabulary(self, attribute: Text) -> Dict[Text, int]:
+        """Get trained vocabulary from attribute's count vectorizer."""
+        return self.vectorizers[attribute].vocabulary_
```

Comment:
can't the attribute be not in `vectorisers`?

Reply:
My philosophy here was that if the code is used in the wrong manner, I'd let it crash early - instead of half-heartedly returning `None` and then not knowing what to do with that or masking the error ...  🤔